Mathlib/CategoryTheory/SmallObject/Iteration.lean

```diff
@@ -172,17 +170,22 @@ instance : Category (Iteration ε j) where
   id := id
   comp := comp
 
-instance : Subsingleton (iter₁ ⟶ iter₂) where
-  allEq f g := ext' (by
-    let P := fun (i : J) => ∀ (hi : i ≤ j), f.natTrans.app ⟨i, hi⟩ = g.natTrans.app ⟨i, hi⟩
-    suffices ∀ (i : J), P i by
+instance {J} {j : J} [ConditionallyCompleteLinearOrderBot J] [WellFoundedLT J] [SuccOrder J]
```

Comment:
Does [OrderBot] not suffice here?

Reply:
Yeah, `PartialOrder` + `OrderBot` instead of `ConditionallyCompleteLinearOrderBot` does the job.